recipes/molgrid/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "molgrid" %}
+{% set version = "0.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gnina/libmolgrid/archive/refs/tags/v0.5.2.tar.gz
+  sha256: e732d13a96c2f374d57a73999119bef700172d392c195c751214aa6ac6680c3a
+
+  patches:
+    - python-install-no-git.patch
+
+build:
+  number: 0
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                                # [cuda_compiler_version == "None"]
+  skip: true  # [osx or win or (cuda_compiler_version in (undefined, 'None')) or cuda_compiler_version == 10.2]
+
+requirements:
+  build:
+    - python
+    - numpy
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - cmake
+    - make  # [unix]
+    - pytest
+  host:
+    - nccl  # [cuda_compiler_version != "None"]
+    - zlib
+    - boost
+    - pytest
+    - numpy
+    - python
+    - openbabel >=3
+    # - pyquaternion
+    - pthread-stubs
+    - eigen
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - openbabel >=3
+    - pytorch >=1.10.2
+    - scipy
+    - psutil
+    # - pyquaternion
+    - boost
+    - boost-cpp
```

Comment:
Should `boost-cpp` also be listed in `host`?

Reply:
Will do.